packages/webview_flutter/webview_flutter_android/android/src/main/java/io/flutter/plugins/webviewflutter/WebViewHostApiImpl.java

```diff
@@ -277,6 +282,57 @@ public void goForward(@NonNull Long instanceId) {
     webView.goForward();
   }
 
+  @Override
+  public void setAuthCredentials(@NonNull Long instanceId, @NonNull String host, @NonNull String realm, @NonNull String username, @NonNull String password, @NonNull GeneratedAndroidWebView.Result<Void> callBack) {
+    WebView webView = Objects.requireNonNull(instanceManager.getInstance(instanceId));
+    WebViewDatabase webViewDatabase = WebViewDatabase.getInstance(webView.getContext());
+
+
+    System.out.println("PROCEED WITH AUTH");
+    WebViewClientFlutterApiImpl.publicHttpHandler.proceed(username, password);
+
+
+//    try{
```

Comment:
why is this code commented out? Can it be removed? And if we need it later, can you add a comment to it when or in what version we need it? Although it looks as if we can remove it. :)